benchmark/BenchmarkExamples.cpp

```diff
@@ -49,19 +51,17 @@ class ConfigOptions : public BenchmarkInterface {
   ConfigOptions() {
     ad_utility::ConfigManager& manager = getConfigManager();
 
-    manager.createConfigOption<std::string>("date", "The current date.",
-                                            &dateString_, "22.3.2023");
+    manager.addOption("date", "The current date.", &dateString_, "22.3.2023"s);
 
-    manager.createConfigOption<int>("numSigns", "The number of street signs.",
-                                    &numberOfStreetSigns_, 10);
+    manager.addOption("numSigns", "The number of street signs.",
+                      &numberOfStreetSigns_, 10);
 
-    manager.createConfigOption<std::vector<bool>>(
-        "CoinFlipTry", "The number of succesful coin flips.", &wonOnTryX_,
-        std::vector{false, false, false, false, false});
+    manager.addOption("CoinFlipTry", "The number of succesful coin flips.",
+                      &wonOnTryX_, {false, false, false, false, false});
 
-    manager.createConfigOption<float>({"Accounts", "Personal", "Steve"},
-                                      "Steves saving account balance.",
-                                      &balanceOnStevesSavingAccount_, -41.9);
+    manager.addOption({"Accounts", "Personal", "Steve"},
+                      "Steves saving account balance.",
```

Comment:
Maybe also use the `s` literal suffix here. 
So then people are less confused, why it doesn't work with two entries, but with three.

Reply:
Done, see [86ee1f0](https://github.com/ad-freiburg/qlever/pull/1040/commits/86ee1f0392b0b90f7e02489d959182bf5807fbff).